### PR TITLE
test for branch protection rule

### PR DESCRIPTION
**Pull request by jialigit**
as title.

### CODE REVIEW

**Review by backwind1233 — APPROVED** [review]
LGTM

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by chenrujun** [comment]
Closing this PR.
Test passed: PR can only be merged after code owner approved.

https://github.com/microsoft/spring-cloud-azure/settings/branch_protection_rules/1708520


image

**Pull request closed without merging**
